services/workflows-service/src/workflow/dtos/workflow-definition-update-input.ts

```diff
@@ -20,4 +20,5 @@ export class WorkflowDefinitionUpdateInput {
   context?: InputJsonValue;
 
   status?: string;
+  resolvedAt?: Date | null;
```

Comment:
Please add the `class-validator` and `@nestjs/swagger` decorators for `resolvedAt`. 🙏

Reply:
It's not relevant for the current use case.

class-validator - The client shouldn't be able to update this property.
Swagger - The property isn't returned in the HTTP response.